01 accordion solution

germakis opened the pull request:
Решил сделать почти pixel perfect на данном этапе. Фактические значения с фигма могут не совпадать.

germakis committed:
Nothing special

germakis committed:
Accordion Solution

germakis committed:
Accordion Solution transition

.accordion__description:last-child не работает
Криво работает транзиция

germakis committed:
nothing

germakis committed:
Paddings, max-width, .accordion__marker,
.accordion, .accordion__description p:last-child.

germakis committed:
Удаление accordion__container

germakis committed:
Merge branch 'master' into 01-accordion-solution

germakis committed:
nothing

germakis committed:
Класс .accordion, paddings при открытом диве итд
по мелочи, практически pixelperfect.

germakis committed:
delete max-width of block in css

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 03-lection1/02-accordion/accordion.css:
```diff
@@ -1 +1,69 @@
 /* Стилизация аккордеона */
+.accordion {
+	max-width: 1000px;
```
удалим

GSemikozov commented on 03-lection1/02-accordion/accordion.css:
```diff
@@ -1 +1,69 @@
 /* Стилизация аккордеона */
+.accordion {
+	max-width: 1000px;
+	margin: 0px auto;
```
марджины на уровне блока не задаем, удалим (исключение только блок .container)

GSemikozov commented on 03-lection1/02-accordion/accordion.css:
```diff
@@ -1 +1,69 @@
+.accordion {
+	max-width: 1000px;
+	margin: 0px auto;
+	margin-bottom: 8px;
```
по структуре отступ снизу нужен между секциями аккордиона, а не между аккордионами)

GSemikozov commented on 03-lection1/02-accordion/index.html:
```diff
@@ -4,15 +4,62 @@
-            </p>
-        </details>
+        <div class="wrapper">
+            <div class="accordion">
```
см коммент выше - все секции надо вложить в один общий div с классом .accordion

germakis replied:
Я уже так делал: 

div class="accordion"
   div class="accordion__item"
   div class="accordion__item"
   div class="accordion__item"
    ...
/div

Решение не принимал. Я уже ничего не понимаю.

GSemikozov replied:
а где это по истории коммитов или переписки в ПР видно? есть подозрение, что все-таки не так было)

GSemikozov replied:
сейчас - набор из 3 независимых аккордионов, у которых по одной секции. А должны быть - 1 аккордион с 3 секциями.

germakis committed:
Не работают margin, непонятно по вложенности
что требуется

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 03-lection1/02-accordion/accordion.css:
```diff
@@ -1 +1,68 @@
+}
+.accordion__description {
+	color: var(--grey-4);
+	margin-bottom: 8px; /* не работает */
```
мне кажется я уже писал по этому поводу, но - еще разок: не работает потому, что марджин у вложенного `p` без какого-либо класса, а обнуляем - у .accordion__description, у которого и так нет отступа

germakis committed:
div class="accordion"
div class="accordion__item"
div class="accordion__item"
div class="accordion__item"
div
margin работает и без класса для p

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.
